Enable modifying timeout of `xql-generic-query` command (#39581)
demisto/content: Packs/Core/ReleaseNotes/3_2_33.md

#### Integrations

##### XQL Query Engine

> - Updated the XQL Query Engine integration to add timeout_in_seconds as an argument to the xdr-xql-generic-query command.

Reviewer: ```suggestion
- Updated the **xdr-xql-generic-query** command with ***timeout_in_seconds***  argument to allow setting the polling timeout of the command.
```
